wp/headless-wp/includes/classes/Integrations/YoastSEO.php

```diff
@@ -162,4 +168,114 @@ public function maybe_override_sitemap_robots_url( $output ) {
 
 		return $output;
 	}
+
+	/**
+	 * Return list of query vars if the request is /yoast/v1/get_head?url and for search URL based on ?s= parameter.
+	 *
+	 * @return boolean|array False if it's not a yoast search rest api request. Search $query_vars if otherwise.
+	 */
+	public function get_yoast_search_query_vars() {
+
+		if ( ! ( defined( 'REST_REQUEST' ) && REST_REQUEST ) ) {
+			return false;
+		}
+
+		$request_uri = isset( $_SERVER['REQUEST_URI'] ) ? sanitize_url( wp_unslash( $_SERVER['REQUEST_URI'] ) ) : '';
```

Comment:
sanitize_url is deprecated. Looks like `esc_url` is the alternative.

Reply:
changed